pages/apply.html

```diff
@@ -79,103 +82,67 @@
       </fieldset>
 
 
-      <fieldset>
+      <fieldset class="veteran-status">
 
-      <h2><legend>Eligibility:</legend></h2>
+        <h2><legend>Veteran Status:</legend></h2>
 
-        <p>
-          Any individual who is not currently a GSA employee or contractor and who is a U.S. Citizen, a non-citizen national of the U.S., or has been admitted to the U.S. for permanent residence and holds a valid green card is eligible to apply.
```

Comment:
@rooneywp Is this still the accurate language? If so, we've lost it in the current version, which asks only for verification that the applicant is a U.S. Citizen or Green Card holder, and we should restore it.